pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/phimanifest/PhiManifestService.java

```diff
@@ -203,13 +204,11 @@ private boolean hasParticipantConsentedToTumor(ElasticSearchParticipantDto parti
      * If the participant has answered the question, it will be returned
      * as a boolean.  If they have not, an empty optional will be returned.
      */
-    private static Optional<Boolean> getAdultParticipantConsentedToTumorAnswer(ElasticSearchParticipantDto participant, String studyGuid) {
+    private static Optional<Object> getAdultParticipantConsentedToTumorAnswer(ElasticSearchParticipantDto participant, String studyGuid) {
```

Comment:
This was the source of the issue Whitney identified.  Instead of interpreting "no answer" to mean "false", this method now returns an empty optional, leaving it to clients to figure out what to do with an empty response.